server/blob/blob-objectstorage/src/main/java/org/apache/james/blob/objectstorage/ObjectStorageBlobsDAO.java

```diff
@@ -49,6 +50,7 @@
 public class ObjectStorageBlobsDAO implements BlobStore {
     private static final Location DEFAULT_LOCATION = null;
     private static final Logger LOGGER = LoggerFactory.getLogger(ObjectStorageBlobsDAO.class);
+    private static final int BUFFERED_SIZE = 256 * 1024;
```

Comment:
Does it even make sense to save such a small data in object storage at all?

I do strongly believe that the answer is no, even more, when we know this corresponds to headers, that are often read... I do believe we need to invest in a compound BlobStore storing stuff in Cassandra when small and in S3 when big.

I'd be pleased to do pair on that when I come in Lyon, I expect significant gains ;-)

Reply:
That's not the point of this ticket, I agree with @mbaechler let's try first to use our object correctly, then see what's missing regarding performances.

Reply:
Sure that's not the point of this ticket thought I think discussing such kind of things is important. Sorry for making suggestions, that from my point of view is kind of basic.
Btw isn't that worth it just to try???

Reply:
If I dismiss it, user wanting to rely only on object storage will still have a hard time.

I think your idea worth the try, and is independent of my change.